src/platform/platform_linux.cpp

```diff
@@ -245,7 +245,7 @@ std::map<std::string, mp::NetworkInterfaceInfo> mp::platform::Platform::get_netw
 
 QString mp::platform::Platform::get_workflows_url_override() const
```

Comment:
Same thing here, does this function need to be called `get_blueprints_url_override()`?

Reply:
Same as above.